### PR TITLE
Continue to fix bugs encountered in daily release

### PR DESCRIPTION
**Pull request by hwdef**
ref:  https://github.com/volcano-sh/volcano/pull/2448#issuecomment-1221362974

error message: https://github.com/volcano-sh/volcano/runs/7898219193?check_suite_focus=true

### CODE REVIEW

File: hack/publish.sh [hunk 1]
```diff
@@ -57,27 +46,6 @@ fi
...
-docker push ${IMAGE_PREFIX}/vc-scheduler:${VOLCANO_IMAGE_TAG}
-
-echo "pushing ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG}"
-docker tag ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG} ${IMAGE_PREFIX}/vc-admission:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-admission:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG}
-
 echo "Generate release tar files"
 cd ${RELEASE_FOLDER}/
 tar -zcvf volcano-${VOLCANO_IMAGE_TAG}-${OSTYPE}.tar.gz *
```
Comment (shinytang6) [inline hack/publish.sh:51]: do you know why we need that?

**Review by shinytang6 — CHANGES_REQUESTED** [review]

File: hack/publish.sh (hunk 1 above)
Comment (hwdef) [reply hack/publish.sh:51]: I am not sure. I guess for [this](https://github.com/volcano-sh/volcano/archive/refs/tags/v1.6.0.tar.gz)

File: hack/publish.sh [hunk 2]
```diff
@@ -57,27 +46,6 @@ fi
 # overwrite the tag name into values yaml
 sed -i "s/latest/${VOLCANO_IMAGE_TAG}/g" ${RELEASE_FOLDER}/helm/chart/volcano/values.yaml
```
[the remark this answers is not in the document]
Comment (hwdef) [reply hack/publish.sh:48]: I added `VOLCANO_IMAGE_TAG` back

**Comment by hwdef** [comment]
image
daily release still failed. @william-wang @jiangkaihua

**Pull request closed without merging**